editors/code/package.json

```
@@ -283,6 +283,14 @@
                     "default": null,
                     "markdownDescription": "Path to rust-analyzer executable (points to bundled binary by default). If this is set, then `#rust-analyzer.updates.channel#` setting is not used"
                 },
+                "rust-analyzer.serverExtraEnv": {
```

```suggestion
                "rust-analyzer.server.extraEnv": {
```

we'll migrate serverPath later

---

Changed 👍🏻